README.md

```diff
@@ -94,22 +94,61 @@ TODO: more integrations will be added below as the [Hydra GSoC project progresse
 
 #### Authenticate frontend to access private content
 
-After initializing the Bridge, you can import `get_token` method from [hydra.js](https://github.com/collective/volto-hydra/tree/hydra.js). This will return the token which you can use to access private content from Plone backend.
```

Comment:
@MAX-786 this assumes you have to have the bridge loaded all the time even when you aren't using the editor. One goal of a decoupled frontend is to make it as fast and as light as possible. So we need to provide a way for them to know when the editor is being used without having to have the bridge js loaded all the time. This is why I suggest its better to pass the token via the url argument and that signifies the editor is now being used and the frontend should load and init the bridge.

Reply:
Yep, I was looking into your previous comments about this. I'll fix it soon.